custom_components/tuya_local/devices/truelife_air_purrifier_p3.yaml

```diff
@@ -0,0 +1,118 @@
+name: Air Purifier
+products:
+  - id: pc3mcvhmyfponsqf
+    manufacturer: "TrueLife"
+    model: "Air Purifier P3 WIFI"
+entities:
+  - entity: fan
+    icon: "mdi:air-purifier"
+    translation_only_key: fan_with_presets
+    dps:
+      - id: 1
+        type: boolean
+        name: switch
+      - id: 3
+        name: preset_mode
+        type: string
+        mapping:
+          - dps_val: AUTO
+            value: smart
+          - dps_val: SLEEP
+            value: sleep
+          - dps_val: MANUAL
+            value: manual
+  - entity: select
+    name: Fan Speed
+    icon: "mdi:fan"
+    category: config
+    dps:
+      - id: 4
+        type: string
+        name: option
+        mapping:
+          - dps_val: LOW
+            value: low
+          - dps_val: MID
+            value: medium
+          - dps_val: HI
+            value: high
+  - entity: sensor
+    class: pm25
+    category: diagnostic
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        unit: ugm3
+  - entity: switch
+    translation_key: ionizer
+    category: config
+    dps:
+      - id: 6
+        type: boolean
+        name: switch
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 7
+        type: boolean
+        name: lock
+  - entity: switch
+    translation_key: uv_sterilization
+    category: config
+    dps:
+      - id: 9
+        type: boolean
+        name: switch
+  - entity: button
+    translation_key: filter_reset
+    category: config
+    dps:
+      - id: 11
+        type: boolean
+        name: button
+  - entity: sensor
+    name: Filter Usage (hours)
```

Comment:
The unit is already present below, it does not need to be repeated in the entity name. Add `class: duration`